frontend/src/metabase/entities/collections/getInitialCollectionId.ts

```diff
@@ -51,7 +51,7 @@ const getInitialCollectionId = createSelector(
     byCollectionIdProp,
     byCollectionIdNavParam,
     byCollectionUrlId,
```

Comment:
I think these are unnecessary but I'm not brave enough to delete them

Reply:
`byCollectionId` will still work because you're passing `{collectionId: id}` to the selector in `_useGetDefaultCollectionId`. But the others depend on router props that won't be there anymore. Is `getInitialCollectionId` used anywhere else?